lib/Anon.js

```diff
@@ -191,7 +193,7 @@ export default function anonymize(dcm) {
             var value = tagValues[id];
             if (typeof value === "string" && value.match(/eE/) === null) {
                 tagValues[id] = value.substr(0, 16);
-                console.warn("Warning: clipped Slice Location value from '" + value + "' to '" + tagValues[id] + "'.");
+                console.warn("Warning: clipped Slice Location value from '".concat(value, "' to '").concat(tagValues[id], "'."));
```

Comment:
We don't target anything that doesn't support backticks - why is `esbuild` generating such bloated code?



Perhaps `esbuild lib/browser.js --bundle --target esnext --sourcemap --outfile=docs/js/main.js`

Reply:
Ahh, I see - we're using `tsc` to compile typescript to js, then using `esbuild` to compile the results.